### PR TITLE
feat(discover): cleanup index

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2019-11-14 at 1 02 15 PM

### CODE REVIEW

**Comment by doralchan** [comment]
Warning: nothing is connected

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx [hunk 1]
```diff
@@ -73,55 +70,33 @@ class EventsV2 extends React.Component<Props> {
...
-
     return (
       <Banner
         title={t('Discover')}
-        subtitle={t('Here are a few sample queries to kick things off')}
+        subtitle={t('Customize your query searches')}
         onCloseClick={this.handleClick}
       >
-        {sampleQueries}
+        <Button>Build a new query</Button>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/eventsV2/index.tsx]: Should this text be wrapped in t()?

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx [hunk 2]
```diff
@@ -73,55 +70,33 @@ class EventsV2 extends React.Component<Props> {
...
       </Banner>
     );
   }
 
+  renderActions() {
+    return (
+      <StyledActions>
+        <StyledSearchBar />
+        <Button priority="primary">Build a new query</Button>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/eventsV2/index.tsx]: Should this text be wrapped in t()?

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx [hunk 3]
```diff
@@ -32,10 +30,9 @@ import SavedQueryButtonGroup from './savedQuery';
 import EventView from './eventView';
 import EventInputName from './eventInputName';
 import {getFirstQueryString} from './utils';
-import {SAMPLE_VIEWS} from './data';
 import QueryList from './queryList';
 
-const DISPLAY_SEARCH_BAR_FLAG = false;
+const DISPLAY_SEARCH_BAR_FLAG = true;
```
Comment (dashed) [inline src/sentry/static/sentry/app/views/eventsV2/index.tsx]: @doralchan search bar looks good to me. Can you please revert this back to `false` prior to merging?

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx (hunk 1 above)
Comment (dashed) [inline src/sentry/static/sentry/app/views/eventsV2/index.tsx]: Would you be able to add a CTA for this button?

Something like

```
// DEFAULT_EVENT_VIEW_V1 is from data.tsx
const eventView = EventView.fromEventViewv1(DEFAULT_EVENT_VIEW_V1);	

const to = {	
    pathname: location.pathname,	
    query: {	
        ...eventView.generateQueryStringObject(),	
    },	
};

<Button
    to={to}	
    onClick={() => {	
        trackAnalyticsEvent({	
            eventKey: 'discover_v2.prebuilt_query_click',	
            eventName: 'Discoverv2: Click a pre-built query',	
            organization_id: this.props.organization.id,	
            query_name: eventView.name,	
        });	
    }}
>...</Button>
```

**Review by dashed — CHANGES_REQUESTED** [review]
The visuals are great! 👍  

I have some minor feedback.

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/index.tsx]: Yep!

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx (hunk 2 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/index.tsx]: 👍

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/index.tsx]: 👍

File: src/sentry/static/sentry/app/views/eventsV2/index.tsx (hunk 3 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/index.tsx]: 👍

**Comment by doralchan** [comment]
Screen Shot 2019-11-14 at 3 30 01 PM
Hiding search and btn even tho they're still there

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**